docs/tutorials/ai/durable-ai-agent/index.md

```diff
@@ -0,0 +1,6078 @@
+The tool takes the customer's email and trip information such as the cost of the flight, the description of the event, the number of days until the invoice is due, and generates a sample invoice for that user showing the details of the flight and the cost.
+The LLM will use this tool to invoice the customer once the customer has confirmed their travel plans.
+This tool can either use the [Stripe API](#optional) if you have an API key configured in your `.env` file, or it will generate a mock invoice if it is unable to detect an API key.
+
+First, get the tool by running the following command to download it from the [companion GitHub repository](https://github.com/temporal-community/tutorial-temporal-ai-agent):
+
+```command
+curl -o find_events_data.json https://raw.githubusercontent.com/temporal-community/tutorial-temporal-ai-agent/main/tools/create_invoice.py
+```
+
+Next, familiarize yourself with the tool by reviewing the code.
+Try to answer the following questions about the code:
+* What customer related verification does the tool do before creating the invoice?
+* What does the tool do if this verification fails?
+
+Once you have finished reviewing the code, test it.
+
+Create another test within the `scripts` directory named `create_invoice_test.py` and add the following code:
+
+```python
+from tools.create_invoice import create_invoice
+
+if __name__ == "__main__":
+
+    args_create = {
+        "email": "ziggy.tardigrade@example.com",
+        "amount": 150.00,
+        "description": "Flight to Replay",
+        "days_until_due": 7,
+    }
+    invoice_details = create_invoice(args_create)
+    print(invoice_details)
+```
+
+However, since this tool can operate in either a mock mode or live API mode, there are two ways to verify it.
+
+#### Testing the mocked `create_invoice` tool
+
+Start by testing it without the Stripe key. 
+If you have it set in your `.env` file, comment it out for now, or skip this step.
+
+Run the test using the following command:
+
+```command
+uv run scripts/search_flight_test.py
+```
+
+The output should be:
+
+```output
+[CreateInvoice] Creating invoice with: {'email': 'ziggy.tardigrade@example.com', 'amount': 150.0, 'description': 'Flight to Replay', 'days_until_due': 7}
+{'invoiceStatus': 'generated', 'invoiceURL': 'https://pay.example.com/invoice/12345', 'reference': 'INV-12345'}
+```
+
+If you aren't planning on using the Stripe API, you can skip this next step and continue if you'd like.
+
+#### Testing the Stripe-powered `create_invoice` tool
+
+Testing the Stripe powered version of the tool is nearly identical to testing the mocked version of the tool.
+
+First, if you haven't uncommented the `STRIPE_API_KEY` lines in your `.env` file and added your API key, do this before running the test.
+
+```bash
+STRIPE_API_KEY=YOUR_STRIPE_API_KEY
+```
+
+:::warning
+
+Make sure you have set up your Stripe account as a [sandbox](https://docs.stripe.com/sandboxes) and are using an API key from there.
```

Comment:
```suggestion
Make sure you have set up your Stripe account as a [sandbox](https://docs.stripe.com/sandboxes) and are using an API key from there. If it is your first time setting up a Stripe account and you haven't added any billing information, this will be the default.
```